src/main/resources/assets/owo/lang/ru_ru.json

```
@@ -0,0 +1,55 @@
+{
+  "text.owo.itemGroup.tab_template": [
+    { "index": 0 },
+    { "text": " > ", "color": "gray" },
+    { "index": 1, "color": "dark_gray" }
+  ],
+  "text.owo.itemGroup.select_hint": {"text": "Шифт-кликните, чтобы выбрать несколько", "color": "gray"},
```

Вот это точно глаза режет.
```suggestion
  "text.owo.itemGroup.select_hint": {"text": "Кликните с зажатым Shift, чтобы выбрать несколько", "color": "gray"},
```